### ISSUE TITLE
Bug: How to debug OnChangePlugin not firing?

### ISSUE THREAD

**Issue by 6peterlu** [facebook/lexical#2470]
I'm following the code sandbox example provided here: https://codesandbox.io/s/lexical-plain-text-example-g932e

and as far as I can tell there are no meaningful differences between the sandbox example and what I'm running locally, but my OnChangePlugin does not fire.

I've replaced the body of the `onChange` function with the following and do not see this in my logs:
```
              onChange={(editorState, editor) => {
                console.log('on change');
              }}
```
The `AutoFocusPlugin` works correctly and I see the blinking cursor. When I type, I do not see any text updates. I'm not setting anything for `initialConfig.editorState`. I did have this working at some point back on the `0.2.1` version but recently updated due to some placeholder formatting issues.

Any help would be appreciated 🙏 

Package versions:
```
"@lexical/react": "0.3.5",
"react": "18.1.0",
"react-dom": "18.1.0"
```

**Comment by 6peterlu** [comment]
Here is a snippet from my working version on old packages:
```
  return (
    <>
      <LexicalComposer initialConfig={initialConfig}>
        <div
          style={{
            paddingLeft: '20px',
            paddingRight: '20px'
          }}
        >
          <LexicalPlainTextPlugin
            placeholder={<TextPlaceholder />}
            contentEditable={
              <LexicalContentEditable
                style={{ outline: 0 }}
              />
            }
            initialEditorState={
              notesRetrieveData.getNote.note.content
            }
          />
          <LexicalOnChangePlugin
            onChange={(editorState) =>
              setNotesState(editorState)
            }
          />
        </div>
      </LexicalComposer>
      <Button
        onClick={async () => {
          await saveNote({
            variables: {
              noteID,
              content: JSON.stringify(notesState)
            }
          });
        }}
        disabled={notesMutationLoading}
      >
        Save
      </Button>
    </>
  );
  
```

packages:
```
"@lexical/react": "0.2.4",
"react":"17.0.0",
"react-dom":"17.0.0",
```

**Comment by 6peterlu** [comment]
The new, not working code:
```
    <>
      <div>
        <LexicalComposer initialConfig={initialConfig}>
          <div
            style={{
              paddingLeft: '20px',
              paddingRight: '20px'
            }}
          >
            <PlainTextPlugin
              placeholder={<TextPlaceholder />}
              contentEditable={
                <ContentEditable style={{ outline: 0 }} />
              }
            />
            <OnChangePlugin
              onChange={(editorState, editor) => {
                console.log('on change');
              }}
              ignoreSelectionChange
            />
            <AutoFocusPlugin />
          </div>
        </LexicalComposer>
      </div>
      <Button
        onClick={async () => {
          await saveNote({
            variables: {
              noteID,
              content: JSON.stringify(notesState)
            }
          });
        }}
        disabled={notesMutationLoading}
      >
        Save
      </Button>
    </>
```

**Comment by always-maap** [comment]
Hey @6peterlu. I tested with the latest versions, and everything was fine. I couldn't reproduce your problem. It would be nice to attach a sandbox of reproduction <3

https://stackblitz.com/edit/react-ts-zgvbdh?file=App.tsx

**Comment by harish-sethuraman** [comment]
I guess may be you missed to fix the import from default import to named import?  (This is what I missed while migrating from older version to new one)

you have to do
`import { OnChangePlugin } from "@lexical/react/LexicalOnChangePlugin";`

**Comment by 6peterlu** [comment]
I figured it out! I had upgraded my `@lexical/react` package but not `lexical`. `lexical` was still stuck at `0.2.4`. After upgrading it to `0.3.5`, everything's working again! Thanks for all the help and my apologies for not including a sandbox. Things were working in the sandbox as you demonstrated so my issue was figuring out what was different between the sandbox and my local haha.

Thanks again @always-maap and @harish-sethuraman 🙏
